doc/api/stream.md

```diff
@@ -1172,6 +1174,14 @@ implementors should not override this method, but instead implement
 [`readable._destroy`][readable-_destroy].
 The default implementation of `_destroy` for `Transform` also emit `'close'`.
 
+#### Class Method: stream.pump(...streams[, callback])
+
+* two or more streams to pipe between
+* optional callback
```

Comment:
These parameters should be formatted according to the style used in the rest of the documentation.

Also, it would be a good idea to either add the function signature here in some way or at the very least describe in the description below what parameters the callback should have.